### ISSUE TITLE
[Docs] Search functionality not obvious for keyboard/screenreader users

### ISSUE THREAD

**Issue by patrickhlauke** [twbs/bootstrap#21851]
The current search functionality for docs relies on users typing in search terms and then selecting matches from the dropdown of results shown.

Visually, the dropdown appears like options should be reachable with cursor up/down - but unintuitively, user has to TAB to them (as they're simply links).

For screenreader users, no announcement is made that results are being dynamically shown.

Lastly, submitting the form/hitting enter in the search field results in the page being reloaded, but no actual search being performed...this is confusing to all users.

**Comment by craigstroman** [comment]
Hi, is this issue being worked on?  If not I would be interested in working on it.

**Comment by craigstroman** [comment]
Do you want me to create a pull request for this?

**Comment by Johann-S** [comment]
Go ahead @craigstroman 👍 every opened​ issue are open for a PR, instead of the issue already have a PR

**Comment by craigstroman** [comment]
Hi, sorry for not responding sooner.  I will create a PR for this tomorrow then.

**Comment by mdo** [comment]
Should be good since #22715.

**Comment by craigstroman** [comment]
Hi,

Glad this was fixed.  

I'm sorry for not completing this sooner like I said I would.  In the past 2 months I actually ended up getting a new job and moving to a new state.

**Comment by mdo** [comment]
No worries! We ended up in a good spot with DocSearch :).

**Comment by patrickhlauke** [comment]
Confirming that this now works a lot better for keyboard/AT users
